website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,244 @@
+that includes:
+
+Setting the `primary_datacenter` parameter on all Consul agents in the primary
+datacenter.  Creating the replication token.  Configuring the
+`primary_datacenter` parameter on all Consul agents in the secondary
+datacenter.  Enabling token replication on the servers in the secondary
+datacenter.  Applying the replication token to all the servers in the secondary
+datacenter. 
+
+This guide is intended to be completed during the initial ACL bootstrapping
+process. 
+
+-> After ACLs are enabled you must have a privileged token to complete any
+operation on either datacenter. The privileged token can be the initial
+`bootstrap` token. 
+
+## Configure the Primary Datacenter
+
+~> Note, if your primary datacenter uses the default `datacenter` name of
+`dc1`, you must set the `datacenter` parameter on the secondary datacenter.
+Otherwise, both datacenters will be named `dc1` and there will be conflicts.
+
+### Consul Servers and Clients
+
+You should explicitly set the  `primary_datacenter` parameter on all servers
+and clients, even though, replication is enabled by default on the primary
+datacenter. Your agent configuration should be similar to the example below.  
+
+```json 
+{ 
+"datacenter": "primary_dc", 
+"primary_datacenter": "primary_dc",
+"acl": { 
+  "enabled": true, 
+  "default_policy": "deny", 
+  "down_policy": "extend-cache",
+  "enable_token_persistence": true 
+  } 
+} 
+```
+
+The `primary_datacenter`
+[parameter](https://www.consul.io/docs/agent/options.html#primary_datacenter)
+sets the primary datacenter to have authority for all ACL information. It is
+important to also set this on the clients, so that the clients can forward API
+requests to the servers. 
+
+Finally, start the agent.
+
+```sh 
+$ consul agent -config-file=server.json 
+```
+
+Complete this process on all agents. 
+
+## Create the Replication Token for ACL Management
+
+Next create the replication token for managing ACLs. The token should be
+created with the following privileges:
+
+acl = "write" which will allow you to replicate tokens.  operator = "read" for
+replicating proxy-default configuration entries.  service_prefix, policy =
+"read" and intentions = "read" for replicating service-default configuration
+entries, CA, and intention data. 
+
+```hcl 
+acl = "write"
+
+operator = "read"
+
+service_prefix "" { 
+	policy = "read" 
+	intentions = "read" 
+} 
+```
+
+Now that you have the ACL rules defined, create a policy. 
+
+```sh 
+$ consul acl policy create -name replication -rules
+@replication-policy.hcl 
+ID:           240f1d01-6517-78d3-ec32-1d237f92ab58
+Name:         replication 
+Description: Datacenters: 
+Rules: acl = "write"
+
+operator = "read"
+
+service_prefix "" { policy = "read" intentions = "read" } 
+```
+
+Finally, use your newly created policy to create the replication token.
+
+```sh 
+$ consul acl token create description "replication token" -policy-name
```

Comment:
same thing as above. Either add a line continuation character or delete the newline